sdk/include/reactos/drivers/bootvid/display.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * PROJECT:     ReactOS video boot driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Provides display settings for BOOTVID, NTOSKRNL and HAL
+ * COPYRIGHT:   Copyright 2020 Dmitry Borisov (di.sean@protonmail.com)
+ */
+
+#pragma once
+
+/* For default VGA */
+#define SCREEN_WIDTH    640
+#define SCREEN_HEIGHT   480
+
+#define VID_SCROLL_AREA_LEFT       32
+#define VID_SCROLL_AREA_TOP        80
+#define VID_SCROLL_AREA_RIGHT     631
+#define VID_SCROLL_AREA_BOTTOM    400
+
+#define VID_PROGRESS_BAR_LEFT     259
+#define VID_PROGRESS_BAR_TOP      352
+
+/* 16px space between shutdown logo and message */
+#define VID_SHUTDOWN_LOGO_LEFT    225
+#define VID_SHUTDOWN_LOGO_TOP     114
+#define VID_SHUTDOWN_MSG_LEFT     213
+#define VID_SHUTDOWN_MSG_TOP      354
+
+#define VID_SKU_AREA_LEFT         418
+#define VID_SKU_AREA_TOP          230
+#define VID_SKU_AREA_RIGHT        454
+#define VID_SKU_AREA_BOTTOM       256
+
+#define VID_SKU_SAVE_AREA_LEFT    413
+#define VID_SKU_SAVE_AREA_TOP     237
+
+#define VID_SKU_TEXT_LEFT         180
+#define VID_SKU_TEXT_TOP          121
+
+#define VID_FOOTER_BGD_TOP        (SCREEN_HEIGHT - 59)
```

Comment:
```suggestion
#define VID_FOOTER_BKG_TOP        (SCREEN_HEIGHT - 59)
```
I think BKG is more common/recognizable abbreviation for background.

Reply:
Or even just "BG", that's even more common. :)